### ISSUE TITLE
Some elements are missing focus rings

### ISSUE THREAD

**Issue by andresgalante** [auth0/cosmos#1179]
Some elements like buttons are missing focus rings.

To fix this issue we should test all focusable items: anchors, form elements, buttons, etc and make sure they all have focus rings.

I highly recommend bringing broswer default's back.
